### DISCUSSION TITLE
Vulnerability disclosure procedure

### DISCUSSION THREAD

**Discussion by cldrn** [naver/ngrinder#863] [Q&A]
Hi,

I would like to know your vulnerability disclosure procedure? I have a couple of things I would like to report privately.

Cheers.

**Comment by cldrn** [comment]
Maybe enabling the security policy would be a good start.

**Comment by donggyu04** [comment]
@cldrn

Can you send the vulnerability to `dong.gyu.lee@navercorp.com`?
thank you for your attention

and we support secure mode.
you can enable this in the `system settings` menu
this prevents script code access to the system.

![image](https://user-images.githubusercontent.com/14273601/163257969-8b16d601-a3bc-431e-8510-530a274084f2.png)
